nixos/modules/services/desktops/pipewire/pipewire.nix

```diff
@@ -72,15 +74,75 @@ in {
           https://github.com/PipeWire/pipewire/blob/master/NEWS
         '';
       };
+
+      config = {
+        defaultConfig = mkOption {
```

Comment:
I don't think we need a to disable default configs? mkForce should be sufficient if one needs to override them?